operator/pkg/ciliumendpointbatch/manager.go

```diff
@@ -43,6 +43,10 @@ type cebTracker struct {
 	ceb *cilium_v2.CiliumEndpointBatch
 	// set of CEPs to be removed in the CEB object in next sync.
 	removedCeps map[string]struct{}
+	// number of CEPs inserted in a CEB
```

Comment:
It would seem more natural to just directly increment metric counter here and it won't need to be reset afterwards.